### PR TITLE
Mobile 7025 migrate oxa banner view

### PR DESCRIPTION
**Pull request by YuriyVelichkoOpenX**
**Description of change**

- Migrate BannerView and Banner View protocol to Swift

### CODE REVIEW

**Review by VadimKhohlovOpenX — COMMENTED** [review]
I see the follow in logs:

Main Thread Checker: UI API called on a background thread: -[UIView setValue:forKey:]
PID: 65024, TID: 6502635, Thread name: (none), Queue name: NSOperationQueue 0x7f8e56f23d40 (QOS: UNSPECIFIED), QoS: 0
Backtrace:
4   PrebidMobileRendering               0x00000001140d8dbc $s21PrebidMobileRendering10MoPubUtilsC13setUpAdObject_8configID13targetingInfo05extraJ06forKeySbSo8NSObjectC_SSSDyS2SGypSgSStFZ + 2540
5   PrebidMobileRendering               0x00000001140a04cf $s21PrebidMobileRendering17MoPubBannerAdUnitC06handleA8Response33_43806915DB459AA1879BFF5486ADC173LL8responseySo06PBMBidJ0C_tF + 911
6   PrebidMobileRendering               0x000000011409fd1b $s21PrebidMobileRendering17MoPubBannerAdUnitC11fetchDemand4with10connection16sdkConfiguration9targeting10completionySo8NSObjectC_So27PBMServerConnectionProtocol_pSo19PBMSDKConfigurationCSo12PBMTargetingCySo08PBMFetchJ6ResultVcSgtFySo14PBMBidResponseCSg_s5Error_pSgtcfU_ + 507
7   PrebidMobileRendering               0x000000011409fddb $sSo14PBMBidResponseCSgs5Error_pSgIeggg_ACSo7NSErrorCSgIeyByy_TR + 139
8   PrebidMobileRendering               0x000000011403af15 __45-[PBMBidRequester requestBidsWithCompletion:]_block_invoke_2 + 2341
9   PrebidMobileRendering               0x0000000113fda671 -[PBMServerConnection processResponse:urlResponse:responseData:error:fullServerCallback:] + 3537
10  PrebidMobileRendering               0x0000000113fd8f96 __62-[PBMServerConnection post:contentType:data:timeout:callback:]_block_invoke + 198
11  CFNetwork                           0x000000010d6ec6ca CFNetwork + 34506
12  CFNetwork                           0x000000010d700992 _CFHTTPMessageSetResponseProxyURL + 17344
13  Foundation                          0x000000010ef39918 __NSBLOCKOPERATION_IS_CALLING_OUT_TO_A_BLOCK__ + 7
14  Foundation                          0x000000010ef39810 -[NSBlockOperation main] + 98
15  Foundation                          0x000000010ef3c82e __NSOPERATION_IS_INVOKING_MAIN__ + 17
16  Foundation                          0x000000010ef38a35 -[NSOperation start] + 785
17  Foundation                          0x000000010ef3d1b3 __NSOPERATIONQUEUE_IS_STARTING_AN_OPERATION__ + 17
18  Foundation                          0x000000010ef3ccc6 __NSOQSchedule_f + 182
19  libdispatch.dylib                   0x00000001169195fb _dispatch_block_async_invoke2 + 83
20  libdispatch.dylib                   0x000000011690a9c8 _dispatch_client_callout + 8
21  libdispatch.dylib                   0x000000011690d316 _dispatch_continuation_pop + 557
22  libdispatch.dylib                   0x000000011690c71c _dispatch_async_redirect_invoke + 779
23  libdispatch.dylib                   0x000000011691c508 _dispatch_root_queue_drain + 351
24  libdispatch.dylib                   0x000000011691ce6d _dispatch_worker_thread2 + 135
25  libsystem_pthread.dylib             0x00007fff6116847a _pthread_wqthread + 244
26  libsystem_pthread.dylib             0x00007fff61167493 start_wqthread + 15

Looks like we need switch to the main thread.

File: PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/BannerView.swift [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+    public let adUnitConfig: PBMAdUnitConfig
+    public let eventHandler: PBMBannerEventHandler?
+    
+    // MARK: - Public Properties
+    
+    public var configID: String {
+        adUnitConfig.configId
+    }
+
+    public var refreshInterval: TimeInterval {
```
Comment (VadimKhohlovOpenX) [inline PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/BannerView.swift]: Do we need @objc modificatory here?

**Review by VadimKhohlovOpenX — COMMENTED** [review]
See my question.

File: PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/BannerView.swift (hunk 1 above)
Comment (YuriyVelichkoOpenX) [reply PrebidMobile/Rendering/PrebidMobileRendering/PrebidMobileRendering/Prebid/Integrations/GAM/BannerView.swift]: @ objc is added

**Review by VadimKhohlovOpenX — APPROVED** [review]
LGTM

**Pull request merged**